src/pages/ImportPage.tsx

```diff
@@ -0,0 +1,756 @@
+  const [showNotification, setShowNotification] = useState<
+    "success" | "error" | ""
+  >("");
+
+  useEffect(() => {
+    setIsLoading(true);
+    const fetchData = async () => {
+      await loadData();
+      setIsLoading(false);
+    };
+    fetchData();
+  }, []);
+
+  const changePaginationState = (pageInfo: {
+    page: number;
+    pageSize: number;
+  }) => {
+    if (page != pageInfo.page) {
+      setPage(pageInfo.page);
+    }
+    if (pageSize != pageInfo.pageSize) {
+      setPageSize(pageInfo.pageSize);
+    }
+  };
+
+  const openModal = (modal: ModalType) => {
+    setOpen(modal);
+  };
+
+  const closeModal = () => {
+    setOpen("");
+  };
+
+  const closeNotification = (
+    _e: React.MouseEvent<HTMLButtonElement, MouseEvent>
+  ) => {
+    setShowNotification("");
+    return true;
+  };
+
+  const loadData = async () => {
+    const data = await getAllData();
+    if (data.length === 0) {
+      setHeaders([]);
+      setRows([]);
+      return;
+    }
+    const fields = Object.keys(data[0]).filter((field) => field !== "id");
+    setHeaders(
+      fields.map((field) => ({
+        header: field,
+        key: field,
+      }))
+    );
+    setRows(data.map((t) => ({ ...t, id: String(t.id) })));
+  };
+
+  const onAdd = async () => {
+    const temp: any = {};
+    headers.forEach((header) => {
+      temp[header.key] = "";
+    });
+    setEditData(temp);
+    openModal("add");
+  };
+
+  const onEdit = async (rowId: string) => {
+    setTargetId(parseInt(rowId));
+    setEditData(await getData(parseInt(rowId)));
+    openModal("edit");
+  };
+
+  const onEditHeader = async () => {
+    const temp: any = {};
+    headers.forEach((header) => {
+      temp[header.key] = header.key;
+    });
+    setEditData(temp);
+    openModal("editHeader");
+  };
+
+  const onDelete = (rowId: string) => {
+    setTargetId(parseInt(rowId));
+    openModal("delete");
+  };
+
+  const onBatchDelete = (selectedRows: readonly DenormalizedRow[]) => {
+    setTargetId(selectedRows.map((row) => parseInt(row.id)));
+    openModal("delete");
+  };
+
+  const removeDataset = () => {
+    setIsLoading(true);
+    clear()
+      .then(async () => {
+        await loadData();
+        setIsLoading(false);
+        setShowNotification("success");
+      })
+      .catch(() => {
+        setIsLoading(false);
+        setShowNotification("error");
+      });
+  };
+
+  return (
+    <div>
+      {showNotification === "success" ? (
+        <ToastNotification
+          kind="success"
+          role="status"
+          timeout={4000}
+          title="Successful"
+          onClose={closeNotification}
+        />
+      ) : showNotification === "error" ? (
+        <ToastNotification
+          kind="error"
+          role="status"
+          timeout={4000}
+          title="An error occurred"
+          onClose={closeNotification}
+        />
+      ) : (
+        <div style={{ height: 50 }}></div>
+      )}
```

Comment:
The code can be written in this way, 

```
{showNotification !== "" &&  (
        <ToastNotification
          kind={showNotification}
          role="status"
          timeout={4000}
          title={showNotification === "sucess" ? "Success" : "an Error Occured" }
          onClose={closeNotification}
        />
      )}
      
```

Reply:
This change has been included in the latest commit.